test/art.js

```diff
@@ -1,6 +1,8 @@
 "use strict";
 
 var clc = require("../");
+var supportsColor = require("../supports-color");
+supportsColor.enable();
```

Comment:
I think we should not apply any setting here. For all test files I'd maintain convention, that tweak is made in test function body as:

```javascript

module.exports = function (t, a) {
  supportsColor.enable()
  try {
    // tests
  } finally {
    supportsColor.auto()
  }
};
```

We may also create such decorator in `test/_lib` folder, and reuse in all tests that need that